src/reducers/ui-state-updaters.js

```diff
@@ -164,3 +165,16 @@ export const setExportDataUpdater = (state, action) => ({
     data: !state.exportData.data
   }
 });
+
+export const addNotificationUpdater = (state, {payload}) => ({
+  ...state,
+  notifications: [
+    ...state.notifications,
```

Comment:
`...(state.notifications || [])`

Reply:
notifications has a default value in the reducer. Do you still want to apply the OR statement?